src/header/mod.rs

```diff
@@ -206,6 +206,11 @@ impl Headers {
         self.data.insert(UniCase(name.into_cow()), Item::new_raw(value));
     }
 
+    /// Remove a header set by set_raw
+    pub fn remove_raw<K: IntoCow<'static, str>>(&mut self, name: K) {
```

Comment:
Since the name isn't going to be stored, we can relax the requirement, just like with `get_raw`. We can accept any &str, not just static ones.